testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeColumnMappingMode.java

```diff
@@ -67,35 +67,140 @@ public void testColumnMappingModeName()
     {
         String tableName = "test_dl_column_mapping_mode_name" + randomTableSuffix();
 
+        onDelta().executeQuery("" +
+                "CREATE TABLE default." + tableName +
+                " (a_number INT, array_col ARRAY<STRUCT<array_struct_element: STRING>>, nested STRUCT<field1: STRING>, a_string STRING)" +
+                " USING delta " +
+                " LOCATION 's3://" + bucketName + "/databricks-compatibility-test-" + tableName + "'" +
+                " TBLPROPERTIES (" +
+                " 'delta.columnMapping.mode'='name'," +
+                " 'delta.minReaderVersion'='2'," +
+                " 'delta.minWriterVersion'='5')");
+
+        try {
+            onDelta().executeQuery("" +
```

Comment:
You need to do ~10 operations on a table for Delta to trigger a Parquet checkpoint. Can we either add more inserts here, or have another copy of the test that hits a checkpoint? Same for the `testShowStatsForColumnMappingModeName` test

Reply:
This comment relates to https://github.com/trinodb/trino/pull/12675#issuecomment-1158114133, right? Could you share how we can reach to the `if` condition? I couldn't go there even if I execute insert statements more than 10.

Reply:
Try creating the table with the properties `delta.checkpoint.writeStatsAsStruct = true, delta.checkpoint.writeStatsAsJson = false`

Reply:
It seems only Databrcricks environment can go to the condition. OSS Delta Lake environment might have limited support for name mapping.

Reply:
> Those `delta.checkpoint.writeStatsAs*` didn't work as far as I tried

Last time i checked, OSS Delta had no support for stats.